tools/management/commands/build_bias_data.py

```diff
@@ -0,0 +1,859 @@
+    cell_cache = dict()
+
+    def add_arguments(self, parser):
+        parser.add_argument('-u', '--purge',
+                            action='store_true',
+                            dest='purge',
+                            default=False,
+                            help='Purge existing bias records')
+
+    def handle(self, *args, **options):
+        # delete any existing structure data
+        if options['purge']:
+            try:
+                print('Started purging bias data')
+                Command.purge_bias_data()
+                print('Ended purging bias data')
+            except Exception as msg:
+                print(msg)
+        print('CREATING BIAS DATA')
+
+        self.build_bias_data()
+        self.logger.info('COMPLETED CREATING BIAS DATA')
+
+    @staticmethod
+    def purge_bias_data():
+        delete_bias_experiment = AnalyzedExperiment.objects.all()
+        delete_bias_experiment.delete()
+
+
+    @staticmethod
+    def process_gproteins_excel():
+        source_file_path = None
+        filenames = os.listdir(Command.structure_data_dir)
+        for source_file in filenames:
+            source_file_path = os.sep.join(
+                [Command.structure_data_dir, source_file]).replace('//', '/')
+            print(source_file, source_file_path)
+        df = pd.read_excel(source_file_path)
+        Command.gprot_cache = df.set_index('UniProt').T.to_dict('dict')
+
+    @staticmethod
+    def process_cell_line_excel():
+        source_file_path = None
+        filenames = os.listdir(Command.cell_structure_data_dir)
+        for source_file in filenames:
+            source_file_path = os.sep.join(
+                [Command.cell_structure_data_dir, source_file]).replace('//', '/')
+            print(source_file, source_file_path)
+        df = pd.read_excel(source_file_path)
+        Command.cell_cache = df.set_index("Cell_line_name").T.to_dict('dict')
+
+    def build_bias_data(self):
+        print('stage # 1, process excell with g_proteins')
+        Command.process_gproteins_excel()
+        Command.process_cell_line_excel()
+        print('Build bias data gproteins')
+        context = dict()
+        content = Command.get_data_from_model()
+        # import pdb; pdb.set_trace()
+        print('stage # 2 : Getting data finished, data points: ', len(content))
+        content_with_children = Command.process_data(content)
+        # import pdb; pdb.set_trace()
+        print('stage # 3: Processing children in queryset finished',
+              len(content_with_children))
+        changed_data = self.queryset_to_dict(content_with_children)
+        # import pdb; pdb.set_trace()
+        print('stage # 4: Converting queryset into dict finished', len(changed_data))
+        send = Command.combine_unique(changed_data)
+        # import pdb; pdb.set_trace()
+        print('stage # 5: Selecting endogenous ligands finished')
+        referenced_assay = self.process_referenced_assays(send)
```

Comment:
This is labeled as "self" but it'a staticmethod. Shouldn't it be a Command call like the others?